scripts/test

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Could you give some info about this line? Thx.

Reply:
Sure. NixOS doesn't _have_ a `/bin/bash`; the only executables with hardcoded locations are `/bin/sh` and `/usr/bin/env`; everything else comes from a generated PATH with pointers into `/nix/store/<hash>-<application>-<version>/bin`.

Using `#!/usr/bin/env bash` causes a PATH lookup to be used to find the shell interpreter.